storage_broker/src/bin/storage_broker.rs

```diff
@@ -682,7 +682,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         builder
             .http2()
             .timer(TokioTimer::new())
-            .keep_alive_interval(Some(args.http2_keepalive_interval));
+            .keep_alive_interval(Some(args.http2_keepalive_interval))
+            .max_concurrent_streams(None);
```

Comment:
According to docs
https://docs.rs/tonic/latest/tonic/transport/struct.Server.html
 None is the default value, so comment would be useful.

Reply:
Indeed, None is the default for the tonic server, but we don't use tonic's server here. Instead, we roll our own so that we can also accept http 1 connections for metrics in parallel.

Reply:
added a comment.